tests/performance/raw_indexing/raw_indexing.rb

```diff
@@ -0,0 +1,101 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance_test'
+require 'app_generator/search_app'
+require 'json_document_writer'
+require 'indexed_search_test'
+require 'performance/fbench'
+require 'pp'
+
+class FeedingIndexTest < PerformanceTest
+    
+    def setup
+        set_owner("onorum")
+        set_description("simple test for indexing and querry preformance")
+        
+        @tainted = false
+        @warmUpDocPath = "#{dirs.tmpdir}warm_up_docs.json"
+        @docPath = "#{dirs.tmpdir}feed_docs.json"
+        
+        deploy_app(create_app)
+        
+        set_up_files
+        
+        start
+    end
+
+    def download_doc_file
+        container = (vespa.qrserver['0'] or vespa.container.values.first)
+        container.execute("cd #{dirs.tmpdir} && python3 #{selfdir}download_webtext.py && cd /")
+    end
+
+    def make_feed_docs
+        container = (vespa.qrserver['0'] or vespa.container.values.first)
+        container.execute("cd #{dirs.tmpdir} && python3 #{selfdir}make_json_docs.py 200000 data/webtext.train.jsonl feed_docs.json && cd /")
```

Comment:
The `cd /` at the end is not needed as these commands are executed in their own sub shells.

Reply:
I can remove them before merging